### DISCUSSION TITLE
windows 10 search box for windows version 21h2 only (or a custom made windows 10 searchbox [Like everythingtoolbar])

### DISCUSSION THREAD

**Discussion by zett1201** [valinet/ExplorerPatcher#2535] [Ideas]
i think that option would be good and work on 21h2 only (except if a different solution was found to make it work on 22h2 and 23h2 like taking the code from the windows 10 searchbox and making it a 1:1 replica of that bar.....) this option can be toggled on if you want and toggled off if you dont want it. this with the windows 10 search menu enabled by default and with the option to enable the windows 11 one... (the windows 10 search menu probably needs to be implemented) plus with the unasked highlight thi-
![Unnamed](https://github.com/valinet/ExplorerPatcher/assets/152609200/dd1cd7f8-82ff-43d0-9231-78fb4daba847)
(this is a concept of how it might look like, yes i put the windows 10 taskbar on top of the image yes...)
![concept](https://github.com/valinet/ExplorerPatcher/assets/152609200/64327079-59ec-4867-b8f5-d9a82e94a559)

**Comment by zett1201** [comment]
and yes i said 21h2 only or (with the time to remake it) a 1:1 recreation of the bar..... (yes i took the image in windows 10 if u wanted to know) (edit: even 21h2 has a search bar its code can be used to make the windows 10 recreation)

**Comment by pyrates999** [comment]
If EverythingToolbar (https://github.com/srwi/EverythingToolbar) does what you want, you should use that.

**Reply by zett1201** [reply]
there is one problem. THE VERSION WITH THE WINDOWS 10 LOOKING THINGS DOESNT WORK ANYMORE!

**Reply by pyrates999** [reply]
Then you should submit an issue there.

**Reply by zett1201** [reply]
they no longer use the windows 10 style
now they only use the new windows 11 style

**Reply by zett1201** [reply]
its pointless for doing a report on a older version and just for style isnt something for a search bar

**Comment by milestprower92** [comment]
> i think that option would be good and work on 21h2 only (except if a different solution was found to make it work on 22h2 and 23h2 like taking the code from the windows 10 searchbox and making it a 1:1 replica of that bar.....) this option can be toggled on if you want and toggled off if you dont want it. this with the windows 10 search menu enabled by default and with the option to enable the windows 11 one... (the windows 10 search menu probably needs to be implemented) plus with the unasked highlight thi- ![Unnamed](https://private-user-images.githubusercontent.com/152609200/287248392-dd1cd7f8-82ff-43d0-9231-78fb4daba847.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzMTc3NDIsIm5iZiI6MTcwMjMxNzQ0MiwicGF0aCI6Ii8xNTI2MDkyMDAvMjg3MjQ4MzkyLWRkMWNkN2Y4LTgyZmYtNDNkMC05MjMxLTc4ZmI0ZGFiYTg0Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxMVQxNzU3MjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05NWIwMmVkMDc1NjBiZGFhYzdkY2I1ZTA0MzA1Yjg4YzYxYzMwMDg1NjgzMWViNDYxNjNlMTkxMmMyOGFjYWU1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Vg6_jCTKCYFWonXf16h_nKgLHc6F-A8F4QGhBiqzWJw) (this is a concept of how it might look like, yes i put the windows 10 taskbar on top of the image yes...) ![concept](https://private-user-images.githubusercontent.com/152609200/287255889-64327079-59ec-4867-b8f5-d9a82e94a559.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzMTc3NDIsIm5iZiI6MTcwMjMxNzQ0MiwicGF0aCI6Ii8xNTI2MDkyMDAvMjg3MjU1ODg5LTY0MzI3MDc5LTU5ZWMtNDg2Ny1iOGY1LWQ5YTgyZTk0YTU1OS5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxMVQxNzU3MjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00OWMzYTRiZjRjZTRiYTdhOTRkYzY5NmZlZTI1N2Y1NGJhNzg5ZTEzMDIyOTUzYTU0MWNhNTExZGVlN2VhNzE4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.In-C1U-oqiQytjG_EIBT3nAtJdD1VRIbCGWz2nvPtds)

is still possible with windows 11 21h2's explorer because the searchbox is still there but there are no plans to add that back at the moment due to the fact that the search icon does the same thing as the search box.

**Reply by zett1201** [reply]
it does the same but a search bar is literally cooler than a button
